examples/docs/en-US/input.md

```diff
@@ -596,26 +596,26 @@ Search data from server-side.
 
 | Attribute      | Description          | Type      | Accepted Values       | Default  |
 | ----| ----| ----| ---- | ----- |
-|type| Same as the `type` attribute of native input, except that it can be `textarea` | string | — | text |
-|value| binding value | string/number| — | — |
-|maxlength| maximum Input text length| number| — | — |
-|minlength| minimum Input text length| number | — | — |
-|placeholder| placeholder of Input| string | — | — |
-|disabled | whether Input is disabled | boolean | — | false |
-|size | size of Input, works when `type` is not 'textarea' | string | large/small/mini | — |
-|icon | icon name | string | — | — |
-|rows | number of rows of textarea, only works when `type` is 'textarea' | number | — | 2 |
-|autosize | whether textarea has an adaptive height, only works when `type` is 'textarea'. Can accept an object, e.g. { minRows: 2, maxRows: 6 }  | boolean/object | — | false |
+|type| Same as the `type` attribute of native input, except that it can be `textarea` | string | - | text |
```

Comment:
I'm having trouble with the dashes.  When I updated from dev, everything came in as `???`.  When I changed them to dashes, they don't look like the dashes that you are using.  I tried to copy/paste from another MD file, but when I run the build, it renders as `?`.  Can you help me out?

Reply:
I'll take care of it.